src/tests/unit-tests.cpp

```diff
@@ -1,6 +1,64 @@
+#include "../app/doctor/doctor.hpp"
+#include "../app/visit/visit.hpp"
 #include "gtest/gtest.h"
 
-TEST(HelloWorld, Hello) {
-    ASSERT_TRUE(true);
+struct AssociationTest : ::testing::Test
```

Comment:
In general, we should avoid putting all tests in single unit-test file.
If we want to keep ALL tests in tests in tests directory (which is a good idea in my opinion, although some other opinions are that we should keep unit tests with classes in src)...
we should name tests files accordingly: by classes in this case.

If in some tests we use a couple of classes interacting together we should name them ModuleTests / Integration tests or whatever (I am not good in test types classification). 
Anyway, we need to establish a well defined naming convention for our tests.